solr/bootstrap/build.gradle

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+apply plugin: 'java-library'
+
+jar {
+  manifest {
+    attributes(
+            'Main-Class': 'org.apache.solr.bootstrap.SolrBootstrap',
+            'Class-Path': '. lib/start.jar jna.jar'
```

Comment:
This is a chance to clean up manual jar naming, actually. The classpath here should be something like this:
"Class-Path": configurations.runtimeClasspath.collect { "lib/${it.getName()}" }.join(' '))
then bootstrap would have accurate coordinates at the exact library versions it needs to launch. Plus -- the "rename" trickery wouldn't be needed anymore in server.gradle.

The name of the output artifact (bootstrap.jar) can be set in this project as well (so that it doesn't have a version in the name). 
jar.archiveName = "boostrap.jar"

Reply:
I tried this, but got

> * What went wrong:
> A problem occurred evaluating project ':solr:bootstrap'.
> > Not allowed to resolve configuration ':solr:bootstrap:runtimeClasspath' at configuration time (https://guides.gradle.org/performance/#don_t_resolve_dependencies_at_configuration_time). Please upgrade your plugins and double-check your gradle scripts (see stacktrace)
>

Reply:
If you get it working, you may in fact push a change to the PR branch